### ISSUE TITLE
NTR: Noncompliant behavior

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#6057]
**Preferred term label:**
Noncompliant behavior

**Synonyms**
Noncompliant 

**Definition (free text, please give PubMed ID)**
An abnormal social behavior that does not conform to or follow the rules, regulations, or advice of others. 
Source: https://askjan.org/limitations/Non-compliant-Behavior.cfm#:~:text=Noncompliant%20behavior%20involves%20behavior%20that,inability%20to%20meet%20specified%20standards.

**Parent term (use hpo.jax.org/app)**
Abnormal social behavior

for Kids First INCLUDE supplement

**Comment by nicolevasilevsky** [comment]
@pnrobinson I can add this term if it looks okay to you

**Comment by pnrobinson** [comment]
In many medical contexts, non-compliance refers to not taking medications as prescribed. It would be good to come up with a different label for this. I am also not convinced that the definition is always 'abnormal', it could use more context. Is this limited to children?

**Comment by nicolevasilevsky** [comment]
Upon discussion on the INCLUDE supplement call, this term is no longer needed.
